test/cctest/node_test_fixture.h

```diff
@@ -113,7 +114,7 @@ class EnvironmentTestFixture : public NodeTestFixture {
       environment_ = node::CreateEnvironment(isolate_data_,
                                              context_,
                                              1, *argv,
-                                             argv.nr_args(), *argv);
+                                             argv.nr_args(), *argv, &state_);
```

Comment:
I get that that might make this a bit trickier, but ideally we would have the options struct initialized from `argv` itself… maybe for another PR

Reply:
Yep, separate PR is ++